lib/msal-browser/src/app/IPublicClientApplication.ts

```diff
@@ -99,6 +101,7 @@ export const stubbedPublicClientApplication: IPublicClientApplication = {
     ssoSilent: () => {	
         return Promise.reject(BrowserConfigurationAuthError.createStubPcaInstanceCalledError());	
     },
+    // eslint-disable-next-line react/no-multi-comp
```

Comment:
Strange. We should figure out whats going on here, this should defintely not be needed.

Reply:
Ignores removed. Thanks!